DemoqaProject/tests/CheckBoxTest.cs

```diff
@@ -0,0 +1,28 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class CheckBoxTest : BaseSeleniumTest
+    {
+        [Test]
+        public void SelectAll()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            Elements elements = new Elements();
+            elements.NavigateToCheckBox();
+
+            Thread.Sleep(3000);
+            elements.Count();
+            Thread.Sleep(3000);
```

Comment:
Please rid of these sleeps, add explicit waits.

Reply:
I just used for debug purposes.